docs/assets/yamlexamples/simple-bridge.yaml

```diff
@@ -1,20 +1,68 @@
-
-apiVersion: eventing.knative.dev/v1
-kind: Broker
+apiVersion: eventing.triggermesh.io/v1alpha1
+kind: MemoryBroker
 metadata:
   name: events
+
 ---
 
-apiVersion: sources.knative.dev/v1
-kind: PingSource
+apiVersion: sources.triggermesh.io/v1alpha1
```

Comment:
We can have our own pingsource.. if you think that would be better for the docs let me know, i will make it for you.

Reply:
I think it could be valuable to have one. But not a top priority.